doc/it.xsemantics.doc/contents/00-Main.html

```diff
@@ -1,3 +1,14 @@
+<!--
```

Comment:
@mor-n4 no, there's no way to have them from Xdoc, but for another Eclipse project no one complained about that... of course we could leave them for the very first round... O:)